daemon/graphdriver/lcow/lcow_svm.go

```diff
@@ -217,17 +217,19 @@ func (svm *serviceVM) hotAddVHDsAtStart(mvds ...hcsshim.MappedVirtualDisk) error
 }
 
 // hotRemoveVHDs waits for the service vm to start and then removes the vhds.
+// The service VM must not be locked when calling this function.
 func (svm *serviceVM) hotRemoveVHDs(mvds ...hcsshim.MappedVirtualDisk) error {
 	if err := svm.getStartError(); err != nil {
 		return err
 	}
-	return svm.hotRemoveVHDsAtStart(mvds...)
-}
-
-// hotRemoveVHDsAtStart works the same way as hotRemoveVHDs but does not wait for the VM to start.
-func (svm *serviceVM) hotRemoveVHDsAtStart(mvds ...hcsshim.MappedVirtualDisk) error {
 	svm.Lock()
 	defer svm.Unlock()
+	return svm._hotRemoveVHDs(mvds...)
+}
+
+// _hotRemoveVHDs removes VHDs from a service VM. When calling this function,
+// the contract is the service VM lock must be held.
+func (svm *serviceVM) _hotRemoveVHDs(mvds ...hcsshim.MappedVirtualDisk) error {
```

Comment:
Using a `_` prefix is a bit odd in golang codestyle

Reply:
@thaJeztah Not unheard of, but what would you rather have if you're requesting an update rather than approving? You weren't really clear at all 😇

Reply:
If you could rename to something without the prefix, that would have my preference :innocent:

Reply:
I agree that a more descriptive name is preferred to the `_` prefix. Maybe `hotRemoveVHDsNoLock` or `NoWait` or something else that describes the difference, although I haven't come up with a great name so far.

Reply:
Updated